draft-packages/popover/docs/Popover.stories.tsx

```diff
@@ -26,212 +24,365 @@ const Container = ({ children }: { children: React.ReactNode }) => (
   <div style={{ minHeight: "150px" }}>{children}</div>
 )
 
-export const DefaultKaizenSiteDemo = () => (
-  <Container>
-    <Popover heading="Default">
-      Popover body that explains something useful, is optional, and not critical
-      to completing a task.
-    </Popover>
-  </Container>
+const TargetElement = ({
```

Comment:
The updates made in this file:
* Using the new `referenceElementRef` prop, which enables the modern popover
* Adding a "Targen element" div, as so the popover can point to something
* Adding a "Legacy Popover" story, since this still needs to function for backwards compatibility
* Add a "overflow: scroll" story